compiler/include/scopeResolve.h

```diff
@@ -53,6 +54,11 @@ Symbol*  lookupAndCount(const char*           name,
                         astlocT** renameLoc = NULL,
                         bool issueErrors = true);
 
+// lookup in the table for a module scope or in the root module for builtins
```

Comment:
I had a bit of trouble understanding this comment so it might need some clarification.  I think what you're saying is "look at module's table or root module/builtins definition point for the symbol", but my brain read the first part as "lookup in the table for a module-scoped symbol or look in the root module for a builtin symbol" as though the first part was missing a word.  I think this is because "for" is used in two different ways in this sentence - "for" as in "the module's table" in the first part and "for" as in "what you're looking for" in the second, but I might be misinterpreting

Reply:
I'll update the comment. It looks at use statements too anyway.